lib/linguist/languages.yml

```diff
@@ -2026,6 +2026,7 @@ HTML+Django:
   tm_scope: text.html.django
   group: HTML
   extensions:
+  - ".j2"
```

Comment:
The first extension in the list is considered the primary and a search of GitHub suggests it's the most popular, but that doesn't necessarily mean it's the primary extension. 

Is `.j2` is definitely the primary extension for "HTML+Django"?

Reply:
didn't know about that, I'll fix it